server.js

```diff
@@ -1,30 +1,72 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+import express from "express"
+import cors from "cors"
+import animalsData from "./data/zoo-animals.json"
+import expressListEndpoints from "express-list-endpoints"
+
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+  res.send(endpoints)
+})
```

Comment:
when you list the endpoints you can add more info to document your API, for example you can filter the data by queries like `predator` and `animalName`, you could add something like this:

```javascript
app.get("/", (req, res) => {
  // Define the documentation object with endpoint details
  const documentation = {
    Welcome: "Welcome to the Zoo Animals API!",
    Endpoints: expressListEndpoints(app).map((endpoint) => {
      return {
        path: endpoint.path,
        methods: endpoint.methods,
        middlewares: endpoint.middlewares,
        description: endpoint.path === "/animals" ? "Get animals data with optional filtering." : "No description provided."
      };
    }),
    QueryParameters: {
      animalName: "Filter animals by name (case-insensitive).",
      predator: "Filter animals by predator status (true/false)."
    }
  };
``` 

This way you document the full usage of your API 👍